### DISCUSSION TITLE
Integrating `--check-creds` flag into `set_pipeline` step

### DISCUSSION THREAD

**Discussion by spire-allyjweir** [concourse/concourse#7181] [Ideas]
Have been using the experimental `set_pipeline` step to great effect. Great work team!

One of the flags we tend to set when configuring pipelines via `fly set-pipeline` is `--check-creds` which verifies that all the variables that the pipeline refers to exist in your secret manager. This has been super helpful and clears up a lot of trial and error bug hunting when bootstrapping a pipeline for the first time.

Is this something we could perhaps add as an optional flag on the `set_pipeline` step? I would be happy to contribute towards this issue if it is something the maintainers would be happy adding.

Thoughts?

**Comment by taylorsilva** [comment]
Sounds like a really nice quality-of-life feature to me :)
